web/src/engine/websites/Team1x1.ts

```diff
@@ -0,0 +1,47 @@
+import { Tags } from '../Tags';
+import icon from './Team1x1.webp';
+import { Chapter, DecoratableMangaScraper, type Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchRequest } from '../FetchProvider';
+
+//TODO: Implement Url setting
+function MangaExtractor(anchor: HTMLAnchorElement) {
+    const id = anchor.pathname;
+    const title = anchor.querySelector('.tt').textContent.trim();
+    return {id, title };
+}
+
+@Common.MangasMultiPageCSS('/series?page={page}', 'div.bs div.bsx a', 1, 1, 0, MangaExtractor)
+@Common.PagesSinglePageCSS('.page-break img')
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+    public constructor() {
+        super('team1x1', `Team X`, 'https://team11x11.com', Tags.Language.Arabic, Tags.Media.Manhua, Tags.Media.Manhwa);
+    }
+    public override get Icon() {
+        return icon;
+    }
+
+    public override ValidateMangaURL(url: string): boolean {
```

Comment:
Can't this be replaced by the `Common.MangaCSS` class decorator?

Reply:
This is made because they often change url without a portal page.
In Hakuneko url is a setting for this website.

Its for accommodating the future "URL" setting that should be done. Should i add a TODO to?